bc_obps/reporting/tests/utils/bakers.py

```diff
@@ -33,6 +34,15 @@ def report_version_baker(**props) -> ReportVersion:
     return version
 
 
+def facility_report_baker(**props) -> FacilityReport:
```

Comment:
We've moved away from bakers & are now using recipes. I added a facility_report recipe in my last PR, so it should be available to use after a rebase on develop & this baker can be removed

Reply:
Removed this baker and it's uses in the rebase!